db/patterns/google_tag_manager.eno

```diff
@@ -6,8 +6,8 @@ alias: google_tag
 
 --- filters
 ||googletagmanager.com/gtm.js^$3p
-||googletagmanager.com/a?^$3p
-||googletagmanager.com/td?^$3p
+||googletagmanager.com/a\?(.+)&gtm=^$3p
```

Comment:
Need to test again, but I think, it needs be expressed like that (which is not a regular expression):

```suggestion
||googletagmanager.com/a?*&gtm=^$3p
```

Or alternatively, enclose it with `/`, so the adblocker will handle it as a regular expression.